Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -549,13 +549,15 @@ def add_attribute(event_id: int = None, internal: bool = False, demisto_args: di
         new_event (MISPEvent): When this function was called from create event command, the attrubite will be added to
         that existing event.
     """
+    args_value = demisto_args.get('value')
+    value = convert_to_list(args_value)
```

Comment:
Lets use `CommonServerPython`'s `argToList` function (see documentation [here](https://xsoar.pan.dev/docs/reference/api/common-server-python#argtolist)) instead of `convert_to_list`.
It's safe, checks for edge cases and can replace the `convert_to_list` function which use unsafe code.
```suggestion
    value = argToList(demisto_args.get('value'))
```